README.md

```diff
@@ -28,6 +28,24 @@ cat msmc_makefile_stdpopsim_patch > msmc/Makefile && cd msmc && make
 cd ../../
 ```
 
+For using `DFE-alpha`, we need to download the program and extra data (5GB) from http://www.homepages.ed.ac.uk/pkeightl/dfe_alpha/download-dfe-alpha.html
+```
+cd ext
+wget -c http://homepages.ed.ac.uk/eang33/dfe_alpha/dfe-alpha-release-2.16.tar.gz
+tar -xvf dfe-alpha-release-2.16.tar.gz
+cat dfe_alpha_makefile_stdpopsim_patch > dfe-alpha-release-2.16/Makefile && cd dfe-alpha-release-2.16 && make
+wget -c https://datashare.ed.ac.uk/bitstream/handle/10283/2730/data.tar.gz?sequence=1&isAllowed=y
+tar -xvf data.tar.gz\?sequence\=1
+cd ../../
+```
+
+For using `polyDFE` and `grapes`, we can download them from their GitHub repositories.
+```
+cd ext
+git clone https://github.com/paula-tataru/polyDFE.git
+wget -c https://github.com/BioPP/grapes/releases/download/v1.1.0/grapes-x86_64-bin-static-1.1.0-1.tar.gz
+tar -xvf grapes-x86_64-bin-static-1.1.0-1.tar.gz
+```
```

Comment:
@andrewkern here are the instructions for installing `DFE-alpha`, `polyDFE` and `grapes`

Reply:
awesome